src/mesh/mesh.hpp

```diff
@@ -341,10 +342,15 @@ class Mesh {
   }
   int GetNumMeshThreads() const { return num_mesh_threads_; }
   std::int64_t GetTotalCells() {
-    auto &mb = block_list.front();
-    return static_cast<std::int64_t>(nbtotal) * mb.block_size.nx1 * mb.block_size.nx2 *
-           mb.block_size.nx3;
+    auto &pmb = block_list.front();
+    return static_cast<std::int64_t>(nbtotal) * pmb->block_size.nx1 *
+           pmb->block_size.nx2 * pmb->block_size.nx3;
   }
+  // TODO(JMM): Move block_size into mesh.
```

Comment:
Is that a todo for later then I suggest to open an issue to keep track or a todo for this PR?

Reply:
Didn't want to handle it in this PR I think it's slightly bigger surgery. I'll open an issue.

Reply:
Done. #305